src/relax/op/op_common.cc

```diff
@@ -38,7 +38,7 @@ Array<TensorStructInfo> GetInputTensorStructInfo(const Call& call, const BlockBu
     if (sinfo == nullptr) {
       ctx->ReportFatal(Diagnostic::Error(call)
                        << op << " requires the input " << op->arguments[i]->name
-                       << " to be Tensor. However, the given one is "
+                       << " to be Tensor. However, the given one has a "
```

Comment:
This is unrelated to this PR, but found the current message confusing during the debugging.